scaper/core.py

```diff
@@ -33,7 +33,7 @@
 EventSpec = namedtuple(
     'EventSpec',
     ['label', 'source_file', 'source_time', 'event_time', 'event_duration',
-     'snr', 'role', 'pitch_shift', 'time_stretch'], verbose=False)
+     'snr', 'role', 'pitch_shift', 'time_stretch'])
```

Comment:
What's the purpose of this edit? Even though it's the default value, I see no harm in setting it explicitly (e.g. in the event that a future update to namedtuple changes the default value to true).

Reply:
They removed it as a parameter in 3.7. See: https://docs.python.org/3/library/collections.html#collections.namedtuple

Reply:
👍